invenio_rdm_records/resources/resources.py

```diff
@@ -116,13 +116,20 @@ def review_delete(self):
     @request_headers
     @request_view_args
     @request_data
-    @response_handler()  # TODO: probably needs to change
+    @response_handler()
     def review_submit(self):
-        """Submit a draft for review."""
+        """Submit a draft for review or directly publish it."""
+        # we pop the value from the payload as it is used only as a service argument
+        # and it is not part of the request payload
+        require_review = False
```

Comment:
we can replace these 3 lines with:
`require_review = resource_requestctx.data.pop("require_review", False)`